### ISSUE TITLE
[kitty] <ESC> do not return to Normal mode

### ISSUE THREAD

**Issue by 10b14224cc** [akinomyoga/ble.sh#454]
**ble version**: 
**Bash version**: 

```
➜  
GNU bash, version 5.2.26(1)-release (x86_64-pc-linux-gnu) [Arch Linux]
ble.sh, version 0.4.0-devel4+b8d63b7e (noarch) [git 2.45.1, GNU Make 4.4.1, GNU Awk 5.3.0, API 4.0, PMA Avon 8-g1, (GNU MPFR 4.2.1, GNU MP 6.3.0)]
starship, version 1.19.0 (rustc 1.78.0 (9b00956e5 2024-04-29) (Arch Linux rust 1:1.78.0-1), 2024-05-15 19:07:12 +00:00)
zoxide, version 0.9.4 (/usr/bin/zoxide)
atuin, version 18.2.0 (/usr/bin/atuin)
locale: LANG=en_US.UTF-8 LC_TIME=en_DK.UTF-8
terminal: TERM=xterm-kitty wcwidth=15.0-west/15.1-2+ri, kitty:0 (1;4000;35)
```

When pressing <ESC>, ble.sh doesn't return to normal mode.

Instead, it prints a Mojibake and stays in insert mode.

It used to work.

**Comment by akinomyoga** [comment]
I'm currently working on this. This is actually a regression caused by a fix for #445, and it is already reported in https://github.com/akinomyoga/ble.sh/issues/445#issue-2302978212.

**Comment by akinomyoga** [comment]
I'm closing this in favor of https://github.com/akinomyoga/ble.sh/issues/445. Let's continue the discussion there.
